tests/support/helpers.py

```diff
@@ -1565,3 +1566,17 @@ def this_user():
     if salt.utils.is_windows():
         return salt.utils.win_functions.get_current_user(with_domain=False)
     return pwd.getpwuid(os.getuid())[0]
+
+
+def dedent(text, linesep=os.linesep):
+    '''
+    A wrapper around textwrap.dedent that also sets line endings.
+    '''
+    linesep = salt.utils.to_unicode(linesep)
+    unicode_text = textwrap.dedent(salt.utils.to_unicode(text))
+    clean_text = linesep.join(unicode_text.splitlines())
+    if unicode_text.endswith(salt.utils.to_unicode('\n')):
```

Comment:
`to_unicode` isn't really necessary here, since the newline character is part of the ASCII set. You can just use `u'\n'` instead.